factory-reset/eos-factory-reset-users

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+if [ -f /etc/adduser.conf ]; then
+    . /etc/adduser.conf
+fi
+
+first_uid=${FIRST_UID:-1000}
+last_uid=${LAST_UID:-59999}
+
+IFS=':'
+while read -r user _ uid _ _ _ _; do
+    if [[ $uid -ge $first_uid && $uid -le $last_uid ]]; then
+        echo "Deleting user $user ($uid)"
+        userdel -r "$user"
```

Comment:
I think it would make more sense to use the `deluser` wrapper that debian/ubuntu use since it's also what AccountsService (and any packages) are setup to use. That actually corresponds correctly with sourcing `/etc/adduser.conf` (rather than `/etc/login.defs` that the lower level shadow utilities `useradd` and `userdel` use).

Reply:
It makes sense to match whatever is used by the UI to create / remove user accounts, so I'm changing it `deluser --remove-home`. But both `userdel -r` and `deluser --remove-home` seem to be doing the exact same thing, with `deluser` looking for the homedir and mail spool by itself and then calling userdel (without `-r`).